include/pros/motors.hpp

```diff
@@ -842,7 +842,7 @@ class Motor {
 	 * ENODEV - The port cannot be configured as a motor
 	 *
 	 * \return The motor's voltage limit in V or PROS_ERR if the operation failed,
-	 * setting errno.
+	 * setting errno.Motor_Group
```

Comment:
```suggestion
	 * setting errno.
```
accidental I presume?

Reply:
Oops yes I'll fix that